lib/clients/AzureClient.py

```diff
@@ -45,7 +45,11 @@ def __init__(self, operation_name, configuration, directory_persistent, director
             msg = 'Could not determine SCSI host number for data volume'
             self.last_operation(msg, 'failed')
             raise Exception(msg)
-        self.instance_location = None
+        self.instance_location = self.get_instance_location(configuration['instance_id'])
```

Comment:
This is getting initialized again at [L133](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/pull/33/files#diff-c6dd8db3e42ffba9915fc66bad10cbd1L133). Would be good to remove it from there.

Reply:
Its needed as L133 failed to get location when snapshot_size is used to create volume in place of instance_persistent_disk size. The requirement to introduce these changes due to failure at L133 only

Reply:
As I understand, we needed to have the location in the initialization rather than in a function, which might/might not be called before you need the location. Hence, what I am saying is, since it is getting init at [L48](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/pull/33/files#diff-c6dd8db3e42ffba9915fc66bad10cbd1R48), we can remove from L133.

Reply:
Made the changes as suggested.